### ISSUE TITLE
Unable to run app.py (ModuleNotFoundError)

### ISSUE THREAD

**Issue by tealnshack** [saharmor/dalle-playground#135]
I am trying to run through local development and I haven't been able to get app.py to run backend URL output. The error that shows up is : 

> from flask_cors import CORS, cross_origin
> ModuleNotFoundError: No module named 'flask_cors'

I did run the command: 'pip install -r requirements.txt' in the virtual environment that I created, but the error persists.

When running 'pip freeze' command, I get the following 
![Screenshot 2023-06-01 115135](https://github.com/saharmor/dalle-playground/assets/70321344/e255c517-cc6d-4cb2-8f3a-ba2c09589a1b)
 
Further, if I try to install flask_cors module, using 'pip install -U flask-cors', the output shows requirement already satisfied.

**Comment by saharmor** [comment]
@tealnshack have you tried running `pip install -U flask-cors`?

**Comment by tealnshack** [comment]
Yeah, when I do it, the package is visible but when I run the command to start the web server the same error pops up despite the package being installed in the virtual environment
